.github/workflows/native-image.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Native Image
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v*'
+
+jobs:
+  native-image:
+    env:
+      LIBSIGNAL_VERSION: 0.17.0
+    strategy:
+      fail-fast: false
+      matrix:
+        job_name:
+          - linux-amd64
+          - macos-amd64
+          #- windows-amd64
+
+        include:
+          - job_name: linux-amd64
+            os: ubuntu-latest
+
+          - job_name: macos-amd64
+            os: macOS-latest
+
+          #- job_name: windows-amd64
+          #  os: windows-latest
+      
+    name: ${{ matrix.job_name }}
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # git checkout
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      # Add libs
+      - name: Add lib on *nix x86
+        run: |
+          curl -Lo /tmp/libsignal_jni.so.tar.gz "https://github.com/exquo/signal-libs-build/releases/download/libsignal-client_v${LIBSIGNAL_VERSION}/libsignal_jni.so-v${LIBSIGNAL_VERSION}-x86_64-unknown-linux-gnu.tar.gz" 
```

Comment:
any reason why exquo's libs are used here and not the official builds, like it's done in https://github.com/AsamK/signal-cli/blob/master/.github/workflows/repackage-native-libs.yml#L51

Reply:
I tried to use the same signal-client builds (even the same job) as from the `repackage-native-libs.yml`. However, this was not properly included in the native image.
-- Reason: I do not know.